common/test/db_fixtures/third_party_auth.json

```diff
@@ -49,6 +51,7 @@
       "icon_image": "test-icon.png",
       "backend_name": "dummy",
       "provider_slug": "dummy",
+      "slug": "dummy",
```

Comment:
Seems like you should remove the old fields here along with this PR.

Reply:
I haven't touched the previous slug fields yet and they are still required to be set - I was planning to remove all references when I remove the old slug fields, but will I have to remove them being set first and make those fields nullable as part of this PR?

Reply:
No, nevermind. I think you have this right. I was just wondering if leaving these fields in the fixture would mask something that we might have missed when moving to the new field, but I see now that the old fields are still required.